src/pages/contributing/translation-program/acknowledgements.tsx

```diff
@@ -249,14 +249,14 @@ const TranslatorAcknowledgements = () => {
         </Flex>
       </Content>
 
-      <Content id="poap">
+      <Content id="oats">
         <ContentHeading
           as="h2"
           fontSize={{ base: "2xl", md: "2rem" }}
           fontWeight={600}
         >
           {t(
-            "page-contributing-translation-program-acknowledgements-poaps-title"
+            "page-contributing-translation-program-acknowledgements-oats-title"
```

Comment:
@lukassim 

<img width="949" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/832d7fe6-72a1-4c2e-9fd2-02a52366cb68">

We end up with a weird case here... The above .json string changes are in the English files only... we changed this string name, but this doesn't exist in other languages yet, so it falls back to the English "OATs"... but the other strings do exists in those langs and have the old copy.

I'd suggest we update this string key in all languages as well, which will display POAP along with everything else in there until these get translated.

Reply:
Good point, though I'd argue OAT shouldn't be translated anyway, same as with POAP
But yeah, there's still mention of POAP in all the subsequent text...

Reply:
One thing to note is that this page doesn't get that many views even in English, and even less in other languages, so I'd also be fine with keeping it this way until the new copy gets translated

Reply:
Hah okay... I just went ahead and fixed that key so the non-English pages will just be consistently not yet updated... the updates will then trickle down in cohesive chunks at least... and like you said, not a lot of eyes.